Dockerfile

```
@@ -1,14 +1,14 @@
 # Using PHP-Apache image
-FROM php:8.0-apache
+FROM php:8.1-apache
 
-# Maintained by Hiob for Galette community
-LABEL maintainer="Hiob <hello@hiob.fr>"
+# Maintained by GrasDK for Galette community
+LABEL maintainer="GrasDK"
 
-LABEL version="0.9.6"
-LABEL description="PHP 8.0 / Apache 2 / Galette 0.9.6"
+LABEL version="1.1.0"
```

I do not know exactly what the label here is for, but you should not yet rely on 1.1 (Galette develop branch).
```suggestion
LABEL version="1.0.0"
```